doc/sphinx/Clusters_from_Scratch/active-passive.rst

```diff
@@ -147,28 +148,30 @@ Verify that pacemaker and corosync are no longer running:
 .. code-block:: none
 
     [root@pcmk-1 ~]# pcs status
-    Error: cluster is not currently running on this node
+    Error: error running crm_mon, is pacemaker running?
+      Could not connect to the CIB: Transport endpoint is not connected
+      crm_mon: Error: cluster is not available on this node
 
 Go to the other node, and check the cluster status.
 
 .. code-block:: none
 
-    [root@pcmk-2 ~]# pcs status
+    [root@pcmk-1 ~]# pcs status
     Cluster name: mycluster
-    Stack: corosync
-    Current DC: pcmk-2 (version 1.1.18-11.el7_5.3-2b07d5c5a9) - partition with quorum
-    Last updated: Mon Sep 10 16:57:22 2018
-    Last change: Mon Sep 10 16:53:42 2018 by root via cibadmin on pcmk-1
-
-    2 nodes configured
-    1 resource configured
-
-    Online: [ pcmk-2 ]
-    OFFLINE: [ pcmk-1 ]
-
-    Full list of resources:
-
-     ClusterIP	(ocf::heartbeat:IPaddr2):	Started pcmk-2
+    Cluster Summary:
+      * Stack: corosync
+      * Current DC: pcmk-2 (version 2.0.5-4.el8-ba59be7122) - partition with quorum
+      * Last updated: Tue Jan 26 19:25:26 2021
+      * Last change:  Tue Jan 26 19:20:28 2021 by root via cibadmin on pcmk-1
+      * 2 nodes configured
+      * 1 resource instance configured
+    
+    Node List:
+      * Online: [ pcmk-2 ]
+      * OFFLINE: [ pcmk-1 ]
+
+    Full List of Resources:
+      * ClusterIP	(ocf::heartbeat:IPaddr2):	 Started pcmk-2
```

Comment:
Unrelated to your changes, I see we overlooked something in the quorum note box below this when we converted to restructured text: the underbars used for emphasis should be asterisks instead. There's also an instance of that in Chapter 7's Prefer One Node Over Another section.